document_url/__manifest__.py

```diff
@@ -3,7 +3,7 @@
 # Copyright 2016 ACSONE SA/NV (<http://acsone.eu>)
 {
     'name': 'URL attachment',
-    'version': '12.0.1.0.0',
+    'version': '12.0.1.1.0',
```

Comment:
Why not letting the bot to change this?

Reply:
I think you have missed the migration script.

Reply:
The migration script must match this, so I felt more secure doing it by hand this time.